modules/op-configuring-default-maximum-scc.adoc

```diff
@@ -0,0 +1,36 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/configuring-security-context-for-pods.adoc
+:_content-type: PROCEDURE
+[id="op-configuring-default-maximum-scc_{context}"]
+= Configuring the default and maximum SCC for pods that {pipelines-shortname} creates
+
+You can configure the default SCC for all pods that {pipelines-shortname} creates. You can also configure the maximum SCC, which is the least restrictive SCC that can be configured for these pods in any namespace.
+
+.Procedure
+
+* Edit the `TektonConfig` custom resource (CR) by entering the following command:
++
+[source,terminal]
+----
+$ oc edit TektonConfig config
+----
++
+Set the default and maximum SCC in the spec, as in the following example:
++
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+#  ...
+  platforms:
+    openshift:
+      scc:
+        default: "restricted-v2" # <1>
+        maxAllowed: "privileged" # <2>
+----
+<1> `spec.platforms.openshift.scc.default` specifies the default SCC that {pipelines-shortname} attaches to the service account (SA) used for workloads (by default, the `pipeline` SA).
```

Comment:
```suggestion
<1> `spec.platforms.openshift.scc.default` specifies the default SCC that {pipelines-shortname} attaches to all TaskRun and PipelineRun pods.
```
i'm suggesting this as the current line could be too much information for the user

Reply:
Can't really agree as the SA is visible and configurable, but I'll add a clarification.